en/docs/ai-gateway/observability/tracing.md

```
@@ -0,0 +1,878 @@
+Linkerd integrates with Jaeger via OpenTelemetry:
+
+```yaml
+tracing:
+  enabled: true
+  endpoint: linkerd-collector.linkerd:4317
+```
+
+## Customizing OpenTelemetry Collector
+
+The OTLP Collector configuration is located at:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)